packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -627,7 +627,7 @@ export class ProjectService extends BaseService {
         return { jobId: schedulerJob.jobId };
     }
 
-    async create(
+    async scheduleCreate(
```

Comment:
These changes are unreadable in Github. Here is a TLDR: 

Split `create` method into 2: `scheduleCreate` and `_create` 

`scheduleCreate` responsibilities:
- creates the legacy job&steps in the database which are used by the UI
- schedule the creation of the project, which will call `_create`

`_create` responsibilities:
- has the same logic as before, tests the adaptor, compiles, etc
- updates the job&steps status